packages/jest-util/src/__tests__/fake_timers.test.js

```diff
@@ -632,26 +491,45 @@ describe('FakeTimers', () => {
       });
 
       timers.runOnlyPendingTimers();
-      expect(runOrder).toEqual(['mock4', 'mock5', 'mock2', 'mock1', 'mock3']);
+      expect(runOrder).toEqual([
+        'mock4',
+        'mock5',
+        'mock2',
+        'mock2',
+        'mock1',
+        'mock2',
+        'mock2',
+        'mock3',
+        'mock1',
+        'mock2',
+      ]);
 
       timers.runOnlyPendingTimers();
       expect(runOrder).toEqual([
         'mock4',
         'mock5',
         'mock2',
+        'mock2',
         'mock1',
+        'mock2',
+        'mock2',
         'mock3',
-
+        'mock1',
+        'mock2',
```

Comment:
Can we extract this (509:518) to `runOrderFirst` and below to `runOrderSecond`? I think we could make it easier to read, as above and lines 495:504 are the same part of `runOrder` queue.

Reply:
Went with


```js
expect(runOrder).toEqual([
  ...firsRunOrder,
  'mock2',
  'mock1',
  'mock2',
  'mock2',
  'mock3',
  'mock5',
  'mock1',
  'mock2',
]);
```

thoughts?